client/src/pages/dashboard/Dashboard.svelte

```diff
@@ -10,19 +10,26 @@
     import { register } from '../register.ts';
 
     let toggleDrawer = false;
+    let selectedId = 1;
+
 </script>
 
+<svelte:head>
+    <link rel="stylesheet" href="/css/dashboard.css" />
+    <title>{`${selectedId}: ${$location.charAt(1).toUpperCase()}${$location.slice(2)}`}</title>

```

Comment:
```suggestion
    <title>{currentPage}</title>

```
Put this reactive statement in `script` like:
```$: currentPage  = `${$location.charAt(1).toUpperCase()}${$location.slice(2)}` ```
So that its not inline in the HTML code. We can omit the `selectedId` to another UI update in the future.